docs/content/rules/sort-astro-attributes.mdx

```diff
@@ -136,6 +136,16 @@ Controls whether sorting should be case-sensitive or not.
 - `true` — Ignore case when sorting alphabetically or naturally (e.g., “A” and “a” are the same).
 - `false` — Consider case when sorting (e.g., “A” comes before “a”).
 
+### specialCharacters
+
+<sub>default: `keep`</sub>
+
+Controls whether special characters should be trimmed, removed or kept before sorting.
+
+- `'keep'` — Keep special characters when sorting (e.g., “_a” comes before “a”).
+- `'trim'` — Trims special characters when sorting alphabetically or naturally (e.g., “_a” and “a” are the same).
```

Comment:
`Trim special...` (to be consistent with `Keep` and `Remove`, or all three with a final `s`).

Obviously it's a very minor remark. Thanks again for your extraordinary work on this repo!